checks/junit.go

```diff
@@ -70,22 +77,49 @@ func (c *JunitChecker) Check(extConfig external.Check) *pkg.CheckResult {
 	if junitCheck.GetDisplayTemplate() != "" {
 		textResults = true
 	}
+	interval := junitCheck.GetInterval()
+	timeout := junitCheck.GetTimeout()
 	var junitStatus JunitStatus
 	template := junitCheck.GetDisplayTemplate()
 	pod := &corev1.Pod{}
 	pod.APIVersion = corev1.SchemeGroupVersion.Version
 	pod.Kind = podKind
+	pod.Labels = map[string]string{
+		junitCheckSelector: junitCheckLabelValue,
+	}
 	if junitCheck.GetNamespace() != "" {
 		pod.Namespace = junitCheck.GetNamespace()
 	} else {
 		pod.Namespace = corev1.NamespaceDefault
```

Comment:
Not sure how this would work? when would the namespace not be defined ?

Reply:
When we are running once-off checks and don't pass the namespace flag so from the code, an empty string would be passed in that case.